starter-workflows/README.md

```diff
@@ -0,0 +1,5 @@
+**Code Scanning Workflows**
+
+GitHub code scanning is a developer-first, GitHub-native approach to easily find security vulnerabilities before they reach production.
```

Comment:
This sentence is not relevant to Frogbot.
```suggestion
```

Reply:
I suggest adding a sentence or two explaining what is in this directory and how to use it.
This may, for example, contain an explanation of how to choose Frogbot in the Stater Workflow, referencing this link: https://docs.github.com/en/actions/using-workflows/using-starter-workflows

Reply:
I changed the text, let me know what you think :)